packages/common/index.ts

```diff
@@ -131,6 +131,8 @@ export class UserPartial {
   @IsInt()
   @IsOptional()
   sid?: number
+
+  userRole?: string
```

Comment:
I think adding this `userRole` is what's causing the tests to fail?

You might need to make a seperate type for this? Maybe it's just missing decorators? Idk